lib/floe/workflow/context.rb

```diff
@@ -21,10 +21,46 @@ def execution
         @context["Execution"]
       end
 
+      def start?
+        execution.key?("StartTime")
+      end
+
+      def end?
+        execution.key?("EndTime")
+      end
```

Comment:
Would `started?` and `running?` make more sense?  `context.running?` I think is a little more obvious than `context.end?`

Reply:
I changed `started?` and `running?` - but left `end?` alone since It is part of the external interface.

Once we change `workflow_instance#run` to `context.running?`, then we can probably drop `context#end?`